src/components/atoms/charts/ContribRadar/index.jsx

```diff
@@ -0,0 +1,186 @@
+//#region > Imports
+//> React
+// Contains all the functionality necessary to define React components
+import React from "react";
+//> Additional
+// Charts for displaying user contribution distribution (Chart.js 2)
+import { Radar } from "react-chartjs-2";
+//#endregion
+
+//#region > Components
+/**
+ * @class A contribution statistic which displays the the ratio of
+ *        commits, issues, pull requests and code reviews.
+ */
+class ContribRadar extends React.Component {
+  state = {
+    dataRadarOptions: {
+      responsive: true,
+      elements: {
+        line: {
+          tension: 0,
+        },
+      },
+      tooltips: {
+        enabled: false,
+      },
+      legend: {
+        display: false,
+      },
+      elements: {
+        point: {
+          radius: 0,
+        },
+      },
+      scale: {
+        pointLabels: {
+          fontSize: 11,
+          color: "grey",
+        },
+        gridLines: {
+          display: false,
+        },
+        ticks: {
+          display: false,
+        },
+      },
+      scales: {
+        yAxes: [
+          {
+            gridLines: {
+              display: false,
+              drawBorder: false,
+            },
+            ticks: {
+              display: false,
+            },
+          },
+        ],
+        xAxes: [
+          {
+            gridLines: {
+              display: false,
+              drawBorder: false,
+            },
+            ticks: {
+              beginAtZero: true,
+              display: false,
+            },
+          },
+        ],
+      },
+    },
+  };
+
+  componentDidMount = () => {
+    // First render chart with current year
+    this.calculateSources(null);
+  };
+
+  componentWillReceiveProps(nextProps) {
+    // Update chart
+    this.calculateSources(nextProps.year);
+  }
+
+  fillChart = (results) => {
+    console.log(results);
```

Comment:
Add newline or remove console.log.
```suggestion
    console.log(results);
    
```